### PR TITLE
[Display_02] Build track data plotter

### LINKED ISSUE #63: [Display_02] Build and Test Display_02 Helper Classes
**What is the use case? Please describe.**
For this sprint task, you'll be building and testing all of the classes in the `edu.oakland.helper.display02` package. Note: This issue is similar to #68 , but is tracked differently as helper and production classes have different deliverable dates.

**Sprint Task Setup**

To be done by @jcortes18 , @sydneyhill3901 , or @Volter43 before the rest of the team starts working on it:

1. Edit this issue message to add a checkbox for each class, and assign it to the correct team member from #52 .

**Prerequisites**

Please makes sure that your test plan from #52 for the class you are writing is closed and merged to main before starting this, to ensure that all of the documentation you need for this is in the correct spot.

**Steps**

To be done by each team member for each class they are developing a test plan for:

1. Each team member should watch the last ~18 minutes of @andrewdimmer 's [Test Driven Development Crash Course](https://youtu.be/eSc0aY4GhPs)
2. One person should create a branch off of main called `define-Name_Of_Class` (ex. the `Demo` classes and tests get build on a branch called `build-Demo`). Everyone else in the programming pair or group should then checkout that branch.
3. Using the information in the video, use test driven development to build all the required classes, stubs, and tests. Remember: you've already defined what you need to build at https://github.com/andrewdimmer/csi-3370-software-project/tree/main/docs/uml-class-diagrams/display02 !
4. Commit your changes with the message `[Display_02] Built Name_Of_Class`
5. Push your changes
6. Create a pull request with the name `[Display_02] Built Name_Of_Class` and the rest of the information like description, tags, etc. For reference, the settings on the right hand side should match this issue, with the exception that the assignees on the PR are just those who are pushing code.
7. Once your code is complete, add @andrewdimmer as a reviewer.

**Tasks to Complete**

- [x] Allocate Team Members to Classes - @jcortes18 , @sydneyhill3901 , or @Volter43
- [ ] Build TrackDataPlotter - @jcortes18 and @sydneyhill3901

**Additional context**
Note: if for some reason you need to change a public interface or shared data type, reach out to @andrewdimmer who can help you manage making that BREAKING CHANGE.

Note: if you run into merge conflicts, let @andrewdimmer know and he can help you resolve them.

### PR DESCRIPTION
**Pull request by sydneyhill3901**
Working on #68

### CODE REVIEW

**Comment by sydneyhill3901** [comment]
Going to work on style changes in the morning, but I see that the build failed because of the libraries we're trying to use. We weren't sure where to include them.

File: src/main/java/edu/oakland/helper/admin/TrackData.java [hunk 1]
```diff
@@ -145,4 +145,4 @@ public String getStatusMessage() {
   public void setStatusMessage(String statusMessage) {
     this.statusMessage = statusMessage;
   }
-}
```
Comment (sydneyhill3901) [inline src/main/java/edu/oakland/helper/admin/TrackData.java:148]: This isn't really a change at all, but I'm not sure how to remove this file from the pull request

Comment (andrewdimmer) [reply src/main/java/edu/oakland/helper/admin/TrackData.java:148]: Just click enter at the end of the file and it should fix itself.

**Comment by sydneyhill3901** [comment]
@andrewdimmer I think our tests are failing because we create a displayable in a headless environment, any ideas on how to handle this? The tests have been changed so that don't call the method to display the chart, but the errors still occur in the tests where the plot gets created.

**Review by andrewdimmer — APPROVED** [review]
Looks good, you are all set to merge!

**Review by jcortes18 — APPROVED** [review]
Thanks Andrew!

**Pull request merged**